plantcv/plantcv/hyperspectral/read_data.py

```diff
@@ -107,7 +107,9 @@ def read_data(filename):
     # Initialize dictionary
     header_dict = {}
 
-    headername = filename + ".hdr"
+    # Remove any file extension and set .hdr filename
+    filename_base = filename.split(".")[0]
```

Comment:
```suggestion
    filename_base = os.path.splitext(filename)[0]
```

This code will be a little more robust in case someone has a file name with more than one "." in it.